### PR TITLE
Add "edit" and "view docs" buttons to subproject list

### LINKED ISSUE #3538: Better UI for editing sub-projects (e.g. "Edit" button)
## Background

See some background discussion in #1341 (relevant comment [here](https://github.com/rtfd/readthedocs.org/issues/1341#issuecomment-358802027)).

## Details

Editing a sub-project is not clear. To edit one, you have to click the title of the project, which is also close to the button to take you to the project page. I was confused at first about how to edit a project and commented in the previous issue, where I found out I was mis-clicking.

An "Edit" button would make this clear and hard to mistake.

## Expected Result

1. Go to sub-project menu
2. See "Edit" button
3. Change canonical URL, name, etc.

## Actual Result

1. Go to sub-project menu
2. Click project title
3. Change canonical URL, name, etc.

### PR DESCRIPTION
**Pull request by stsewd**
Closes #3538 

## Before

screenshot-2018-2-1 subprojects read the docs 1

## After

screenshot-2018-2-1 subprojects read the docs

### CODE REVIEW

**Comment by humitos** [comment]
I like what @stsewd propose here since it's like the common UX all around RTD.

Regarding seeing the URL, I don't have strong thoughts on that. I'll probably just hover the "View Docs" link in that case and check the URL in the bottom of my Firefox. I didn't use too much subprojects but the importance on this (show/hide that URL) will depend on the use case.

I wouldn't use it, I think :)

**Comment by agjohnson** [comment]
Parts of this UI are a move to standardize on our UX, so some notes:

* A URL was added to show the hierarchy of how projects were added. Users weren't adding subprojects correctly and were confused by hierarchy. This makes that UX more clear
* We're moving away from multiple buttons in table rows. They breaks in fantastic ways for other languages. Multiple buttons is also not clear UI, as there is no visual distinction between elements. Secondary actions or options should be a text link -- view docs in this case. 
* Multiple line tables cells will be the pattern in the future for all use cases like this, so i'd rather stick with that, not merge back down to one line.
* Icon only buttons in tables is ideal

The project import page is an example of what other tables will become: multiple lines if necessary, one primary button that is an icon, subtle text hints for secondary information.

The subproject page was a first pass at this feature and no attempt was made to add styling then. Each row does need an edit button, but I think this is dropping some necessary pieces.

**Comment by stsewd** [comment]
@agjohnson I have improved the styles and the layout as suggested, please tell me what you think.

screenshot-2018-2-10 subprojects read the docs 1
screenshot-2018-2-10 subprojects read the docs
screenshot-2018-2-10 subprojects read the docs 2

**Comment by Blendify** [comment]
No the edits to the css file are correct. It looks like they are not generated from a sass file. Maybe something we want to do in the future?

**Comment by stsewd** [comment]
@humitos as @Blendify mention, this isn't an auto generated file. A review for the css from @agjohnson would be awesome here, since I just based on some previous styles.

**Comment by humitos** [comment]
> Maybe something we want to do in the future?

I suppose that this is the path to follow, but I don't have the knowledge to have an opinion :)

**Comment by stsewd** [comment]
I have improve the layout and styles to reuse them on other lists. Hopefully, I didn't break other styles and looks the same!

screenshot-2018-2-21 subprojects read the docs 2
screenshot-2018-2-21 subprojects read the docs 1

File: readthedocs/templates/projects/projectrelationship_list.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 {% block project_edit_content_header %}{% trans "Subprojects" %}{% endblock %}
 
 {% block project_edit_content %}
-  <p>
+  <p class="help_text">
```
Comment (stsewd) [inline readthedocs/templates/projects/projectrelationship_list.html]: This was missing here :)

**Comment by ericholscher** [comment]
Tested this locally and it's 💯.

**Pull request merged**